docs/source/configuration/environmentvariables.md

```diff
@@ -89,3 +89,24 @@ It displays the errors of the non-compliant customizations (in server console) i
 #### Internationalization errors (i18n)
 
 It will enable the log of missing i18n messages (in console).
+
+### Add add-ons via environment variable
+
+Quite often, you need different configurations and enabling components (for example, testing purposes).
```

Comment:
Additional add-ons can be added via `ADDONS` environment variable. This sentence is promising more. Correct me if I'm wrong.

Reply:
The concepts in this paragraph are still not clear to me either. I merely corrected English grammar and syntax, and did not bother with clarity.

Reply:
How about now?

Reply:
Sorry, I don't see any changes. Did you push your commits? I also made some suggestions that were not committed.

Reply:
I committed everything was in https://github.com/plone/volto/pull/3067/commits/b15a8077d2b348e1b51616f252a549d3c37ff9c7

The rewording @ksuess was requesting in https://github.com/plone/volto/pull/3067/commits/11402001c335f2ab1fc9dde776fa0a8f85f6b042